.github/workflows/codacy.yml

```
@@ -48,7 +48,7 @@ jobs:
 
       # Execute Codacy Analysis CLI and generate a SARIF output with the security issues identified during the analysis
       - name: Run Codacy Analysis CLI
-        uses: codacy/codacy-analysis-cli-action@d28ce580f19309cd493628eba4641f745822082c
+        uses: codacy/codacy-analysis-cli-action@d28ce580f19309cd493628eba4641f745822082c # v4.5.0
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)